### DISCUSSION TITLE
How can i reopen an Issue that was wrongly closed?

### DISCUSSION THREAD

**Discussion by guigirl42** [Floorp-Projects/Floorp#1077] [Q&A]
https://github.com/Floorp-Projects/Floorp/issues/1024 was Closed by the Dev after releasing `v11.11.2`. Unfortunately though, that new/current version also introduced a [new feature, which when used, still breaks the URL popups](https://github.com/Floorp-Projects/Floorp/issues/1024#issuecomment-2018043539). That is, the Issue remains relevant, as it is not solved after all, but i fear will get no more attention unless it is changed from Closed to Open. I don't know how i can reopen it.

**Comment by CutterKnife** [comment]
If it is a problem caused by another cause, a new issue can be created.
[In any case, developers will be slow to respond.](https://www.reddit.com/r/Floorp/comments/1bupxiz/developer_is_so_busy_very_busy_has_lots_if_thing)

**Reply by guigirl42** [reply]
T/y for the fast reply! Ah, ok, so Closing is an unrecoverable action, fair enough. Righteo then, i shall open a new issue for it. Yes, i was aware of that thread, & realise that Floorp is going to be _frozen in time_ for a while now [albeit i trust it will still receive all the usual ESR patches, otherwise it will become dangerous to use].

**Reply by creeper-0910** [reply]
Rest assured, the ESR patch will be applied.

**Reply by CutterKnife** [reply]
Another maintainer is continuing development and Sura is auditing the code :)

**Reply by guigirl42** [reply]
Thank you both for that advice. 

Just want to say a big **THANKS** and **CONGRATULATIONS** to all of you responsible for this wonderful project. It's a magnificent gift to the community :heart:
